sdk/tests/core/test_az_mqtt5_rpc_client_codec.c

```diff
@@ -199,9 +198,34 @@ static void test_az_mqtt5_rpc_client_codec_get_publish_topic_buffer_size_failure
       AZ_ERROR_NOT_ENOUGH_SPACE);
```

Comment:
suggestion: assert that test_publish_topic_out_size is updated to the correct value?